### PR TITLE
test: added tests for logging wrapper

### PR DESCRIPTION
**Pull request by phbelitz**
Adds missing tests for the logging wrapper

## Checklist

- [x] PR is rebased to/aimed at branch `develop`
- [x] PR follows [Contributing Guide](../docs/CONTRIBUTING.md)
- [x] Added tests (if necessary)

### CODE REVIEW

File: connaisseur/logging_wrapper.py [hunk 1]
```diff
@@ -11,7 +11,10 @@ def __init__(self, app, log_level):
         self.logger = logging.getLogger(
             "connaisseurRequestLogger"
         )  # has no handler and hence propagates logs to root logger
-        self.logger.setLevel(log_level)
+        try:
+            self.logger.setLevel(log_level)
+        except ValueError:
```
Comment (Starkteetje) [inline connaisseur/logging_wrapper.py:16]: Would you want to log this misconfig?

**Review by Starkteetje — COMMENTED** [review]
See comment, otherwise lgtm

File: connaisseur/logging_wrapper.py (hunk 1 above)
Comment (phbelitz) [reply connaisseur/logging_wrapper.py:16]: hm ... I see now that this is dead code ... a wrong log level already raises an exception in connaisseurs `__main__.py` ... so i guess i revert this back

File: tests/test_logging_wrapper.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+from tracemalloc import start
```
Comment (Starkteetje) [inline tests/test_logging_wrapper.py:1]: I don't see start used, is it a superfluous import?

Comment (Starkteetje) [reply tests/test_logging_wrapper.py:1]: And also, import order is meh

Afaik PEP says python modules emptyline external modules emptyline modules from current project

**Review by Starkteetje — APPROVED** [review]

**Review by Starkteetje — APPROVED** [review]

**Pull request merged**